easybuild/easyconfigs/l/lftp/lftp-4.9.2-GCC-10.2.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'lftp'
+version = '4.9.2'
+
+homepage = 'http://lftp.yar.ru'
+description = """LFTP is a sophisticated ftp/http client, and a file transfer program supporting
+  a number of network protocols. Like BASH, it has job control and uses the readline library for
+  input. It has bookmarks, a built-in mirror command, and can transfer several files in parallel.
+  It was designed with reliability in mind."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://lftp.yar.ru/ftp/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['a37589c61914073f53c5da0e68bd233b41802509d758a022000e1ae2076da733']
+
+dependencies = [
+    ('ncurses', '6.2'),
+#    ('OpenSSL', '1.1.1h'),
```

Comment:
Does it use OpenSSL in any way?
If so, for 10.2.0, we should still use the old
```
osdependencies = [OS_PKG_OPENSSL_DEV]
```
(For newer toolchains we have  the OpenSSL wrapper package)

if not, remove the dependency